Sources/Datadog/RUM/RUMTelemetry.swift

```diff
@@ -156,3 +214,57 @@ internal final class RUMTelemetry: Telemetry {
         }
     }
 }
+
+private extension FeaturesConfiguration {
+    func asTelemetry() -> TelemetryConfigurationEvent.Telemetry.Configuration {
+        let performancePreset = self.common.performance
+        return TelemetryConfigurationEvent.Telemetry.Configuration(
+            actionNameAttribute: nil,
+            batchSize: performancePreset.minUploadDelay.toInt64Milliseconds,
+            batchUploadFrequency: performancePreset.minUploadDelay.toInt64Milliseconds,
+            defaultPrivacyLevel: nil,
+            forwardConsoleLogs: nil,
+            forwardErrorsToLogs: nil,
+            forwardReports: nil,
+            initializationType: nil,
+            mobileVitalsUpdatePeriod: self.rum?.vitalsFrequency?.toInt64Milliseconds,
+            premiumSampleRate: nil,
+            replaySampleRate: nil,
+            sessionReplaySampleRate: nil,
+            sessionSampleRate: self.rum?.sessionSampler.samplingRate.toInt64(),
+            silentMultipleInit: nil,
+            telemetryConfigurationSampleRate: nil,
+            telemetrySampleRate: self.rum?.telemetrySampler.samplingRate.toInt64(),
+            traceSampleRate: self.urlSessionAutoInstrumentation?.tracingSampler.samplingRate.toInt64(),
+            trackBackgroundEvents: self.rum?.backgroundEventTrackingEnabled,
+            trackCrossPlatformLongTasks: nil,
+            trackErrors: self.crashReporting != nil,
+            trackFlutterPerformance: nil,
+            trackFrustrations: self.rum?.frustrationTrackingEnabled,
+            trackInteractions: self.rum?.instrumentation?.uiKitRUMUserActionsPredicate != nil,
+            trackNativeErrors: nil,
+            trackNativeLongTasks: self.rum?.instrumentation?.longTaskThreshold != nil,
+            trackNativeViews: self.rum?.instrumentation?.uiKitRUMViewsPredicate != nil,
+            trackNetworkRequests: self.urlSessionAutoInstrumentation != nil,
+            trackSessionAcrossSubdomains: nil,
+            trackViewsManually: nil,
+            useAllowedTracingOrigins: nil,
+            useAttachToExisting: nil,
+            useBeforeSend: nil,
+            useCrossSiteSessionCookie: nil,
+            useExcludedActivityUrls: nil,
+            useFirstPartyHosts: !(self.rum?.firstPartyHosts.isEmpty ?? true),
+            useLocalEncryption: self.common.encryption != nil,
+            useProxy: self.common.proxyConfiguration != nil,
+            useSecureSessionCookie: nil,
+            useTracing: self.tracing != nil,
+            viewTrackingStrategy: nil
+        )
+    }
+}
+
+private extension Float {
+    func toInt64() -> Int64 {
+        return Int64(self)
+    }
+}
```

Comment:
nit/ We use it to convert user input (sampling rate), so I wonder if this can ever overflow causing a crash `5s` after app starts. Seems safe, [as `Sampler` does sanitization and caps the value](https://github.com/DataDog/dd-sdk-ios/blob/e34d9e69f713e209ce4af89a92a059a7b9810b74/Sources/Datadog/Core/Utils/Sampler.swift#L15).

For sanity, we could use [this helper](https://github.com/DataDog/dd-sdk-ios/blob/e34d9e69f713e209ce4af89a92a059a7b9810b74/Sources/Datadog/Utils/SwiftExtensions.swift#L89) and do:
```swift
func toInt64() -> Int64? {
   return try? Int64(withReportingOverflow: self)
}
```
Better to not have the value than crash.

Reply:
Yeah I'll add in that change.